contracts/liquidity_hub/fee_collector/src/tests/integration.rs

```diff
@@ -3451,6 +3456,14 @@ fn collect_and_distribute_fees_with_expiring_epoch_successfully() {
         }],
     )
     .unwrap();
+
+    // advance the time to one day after the first epoch was created
+    app.set_block(BlockInfo {
+        height: 123456789u64,
+        time: Timestamp::from_nanos(1678888800_000000000u64),
```

Comment:
yeah I am with you, see my comment on why we use nanos everywhere. Though for this particular case, since it's a test, it really doesn't matter, it could be from nanos or from seconds.